test/e2e/test/backend.test.js

```diff
@@ -1,25 +1,54 @@
-
 import { SERVER_URL } from "../config.js";
 import { DEMO_USER } from "../roles.js";
+import { resetDB } from "../../util/testUtils";
 import PageBackend from "../page-objects/PageBackend.js";
+import {GUEST} from "../roles";
 
 fixture("Teste die Seite 'Backend'")
-    .page(SERVER_URL);
-
-
-test('erstelle eine neue Ausstellung', async t => {
-    // step 1 - generate values
-    const uniqueName = "Ausstellung_" + (new Date().getTime());
+    .page(SERVER_URL)
+    .beforeEach(async t => {
+        await t.useRole(DEMO_USER);
+    })
+    .afterEach(async t => {
+        await t.useRole(GUEST);
+    });
 
-    // step 2 - perform steps
-    await t.useRole(DEMO_USER);
+export async function createAusstellung(uniqueName, t) {
     let page = await PageBackend(t).navigateToViaNavbar();
     let tabAustellungen = await page.tabs.chooseAusstellungen();
     let dialog = await tabAustellungen.clickNeueAustellung();
     await dialog.setTitel(uniqueName);
     await dialog.setBeschreibung("Ein paar Worte zur Ausstellung...");
     await dialog.clickAnlegen();
+
+    return tabAustellungen;
+}
+
+test('erstelle eine neue Ausstellung', async t => {
+    // step 0 - initialisierung
+    resetDB();
```

Comment:
Could we move this line into the `beforeEach`-hook since it is needed for both test cases and I guess for all coming test cases ;-)

Reply:
What if we want to run user journey tests?

Reply:
Hm. I am not sure how to implement user journey tests with test cafe at all. In either case (resetting the database in before each or within the tests itself) we run into problems when combing test cases. I think user journey tests will just be additional test cases which access our testing infrastructure (e.g. page objects).

However, we should consider these things when we are at the point to implement user journey tests. Right now it does not matter if where we place the resetDB call. So I will merge it in as it is right now ;-)